lib/jeff.ex

```diff
@@ -85,4 +85,31 @@ defmodule Jeff do
   def set_com(acu, address, params) do
     ACU.send_command(acu, address, COMSET, params).data
   end
+
+  @doc """
+  Send file data to a PD
+  """
+  @spec file_transfer(acu(), osdp_address(), binary()) ::
+          Reply.FileTransferStatus.t() | Reply.ErrorCode.t()
+  def file_transfer(acu, address, data) when is_binary(data) do
+    with %{name: PDCAP, data: caps} <- ACU.send_command(acu, address, CAP) do
+      max = caps[:receive_buffer_size] || 128
```

Comment:
OSDP spec says you can send the `receive_buffer_size` that the reader reports from the capabilities, but I found that it doesn't work. The only size consistently finishing was using `128` bytes